framework/src/st_things/things_stack/framework/things_security_manager.c

```diff
@@ -772,7 +773,7 @@ const unsigned char g_regional_root_ca[] = {
 };
 
 #ifdef CONFIG_ST_THINGS_STG_MODE
-const unsigned char g_regional_test_root_ca[] = {
+unsigned char g_regional_test_root_ca[] = {
```

Comment:
Why did you remove const keyword?
i think we had added const keyword for reducing memory size.

Reply:
ok.